exec/cnex/global.c

```diff
@@ -88,15 +96,27 @@ void global_callFunction(const char *pszFunc, struct tagTExecutor *exec)
     fatal_error("global_callFunction(): \"%s\" - invalid or unsupported predefined function call.", pszFunc);
 }
 
-void print(TExecutor *exec)
+void neon_chr(struct tagTExecutor *exec)
```

Comment:
Inconsistency: Using "struct tagTExecutor" here, and just "TExecutor" in other places.